OpenRA.Mods.Cnc/ImportTiberianDawnLegacyMapCommand.cs

```diff
@@ -0,0 +1,149 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2016 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation. For more information,
+ * see COPYING.
+ */
+#endregion
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using OpenRA.Mods.Common.FileFormats;
+using OpenRA.Mods.Common.UtilityCommands;
+using OpenRA.Primitives;
+
+namespace OpenRA.Mods.Cnc.UtilityCommands
+{
+	class ImportTiberianDawnLegacyMapCommand : ImportLegacyMapCommand, IUtilityCommand
+	{
+		public string Name { get { return "--import-td-map"; } }
+
+		[Desc("FILENAME", "Convert a legacy Tiberian Dawn INI/MPR map to the OpenRA format.")]
+		public override void Run(ModData modData, string[] args)
+		{
+			MapSize = 64; // NOTE: original engine limitation
```

Comment:
It would be better to set this in the constructor: define `public ImportTiberianDawnLegacyMapCommand() : base(64) { }` and then `public ImportLegacyMapCommand(int mapSize) { /* set constant */ }` in the base class.